scripts/pre-commit

```diff
@@ -17,6 +14,29 @@ exit_() {
     exit 1
 }
 
+if [[ $(git config --get user.name | wc -w) -lt 2 ]]; then
+    # This heuristic avoids bad user names such as "root" or "Ubuntu"
+    # or a computer login name. A full name should (usually) have at
+    # least two words. We can change this if needed later.
```

Comment:
So far we don't have any single name contributors, so this is fine with me. We can change it if that situation changes, but given that single name individuals are rare we probably can just make exceptions for them individually.